app/components/views/GetStartedPage/WalletSelection/CreateWalletForm.js

```diff
@@ -2,6 +2,7 @@ import { FormattedMessage as T, defineMessages } from "react-intl";
 import { TextInput } from "inputs";
 import { KeyBlueButton, InvisibleButton, WatchOnlyWalletSwitch } from "buttons";
```

Comment:
Add TrezorWalletSwitch to the index.js in buttons/ so then you can just add the import here instead of seperately below.